draft-iab-use-it-or-lose-it.md

```diff
@@ -212,52 +213,14 @@ broke when new values of the signature_algorithms extension were introduced.
-is immediately apparent.  A two-party protocol has two ends, but even at the
-endpoints of an interaction, protocol elements can be passed on to other
-entities in ways that can affect protocol operation.
-
 One of the key challenges in deploying new features is ensuring compatibility
-with all actors that could be involved in the protocol.
-
-Protocols deployed without active measures against intermediation will tend to
-become intermediated over time, as network operators deploy middleboxes to
-perform some function on traffic {{?PATH-SIGNALS=RFC8558}}.  Any element on path
-can observe unencrypted protocol elements and modify unauthenticated protocol
-elements.
-
-For example, HTTP was specifically designed with intermediation in mind,
-transparent proxies {{?HTTP=I-D.ietf-httpbis-semantics}} are not only possible
-but sometimes advantageous, despite some significant downsides.  Consequently,
-transparent proxies for cleartext HTTP were once commonplace.  Similarly, the
-DNS protocol was designed with intermediation in mind through its use of caching
-recursive resolvers {{?DNS=RFC1034}}.  What was less anticipated was the forced
-spoofing of DNS records by middleboxes such as those that inject authentication
-or pay-wall mechanisms as an authentication and authorization check, which are
-now prevalent in hotels, coffee shops and business networks.
-
-Middleboxes are also protocol participants, to the degree that they are able to
-observe and act in ways that affect the protocol.  The degree to which a
-middlebox participates in a protocol can vary depending on the purpose and
-implementation of the middlebox.  Middleboxes can also apply their own policy,
-with or without the knowledge or permission of endpoints.  For example, a SIP
-back-to-back user agent (B2BUA) {{?B2BUA=RFC7092}} can be very deeply involved
-in the SIP protocol and is often relied upon to provide policy-based security
-controls.
-
-This phenomenon appears at all layers of the protocol stack, especially when
-protocols are not designed with middlebox participation in mind. Some of the
-examples in {{examples}} demonstrate this problem.
-
-By increasing the number of different actors involved in any single protocol
-exchange, the number of potential implementation bugs that a deployment needs to
-contend with also increases.  In particular, incompatible changes to a protocol
-that might be negotiated between endpoints in ignorance of the presence of a
-middlebox can result in a middlebox interfering in negative and
-unexpected ways.
-
-Unfortunately, middleboxes can considerably increase the difficulty of
-deploying new versions or other changes to a protocol.
+with all actors that could be involved in the protocol.  Even the most
+superficially simple protocols can often involve more actors than is immediately
+apparent.  Even for a two-party protocol, protocol elements can be passed on to
+other entities in ways that can affect protocol operation.
```

Comment:
I think it would be good to keep a note about the fact that protocols can be intermediated even though the protocol designers didn't plan for it. Perhaps something like `Protocol designers should consider the potential of their protocol to be intermediated, and account for possible multiple parties in their design.`